### ISSUE TITLE
non standard version number in release

### ISSUE THREAD

**Issue by cmatKhan** [BergmannLab/PascalX#35]
I'm working on the bioconda recipe this morning, which is the origin of this issue:

bioconda doesn't like versions that start with a `v`.

https://bioconda.github.io/contributor/linting.html#version-starts-with-v

They cite PEP 386, which itself says that it has been superseded by [PEP 440](https://peps.python.org/pep-0440/). That PEP says that it is deprecated in favor of a [page on PyPI](https://packaging.python.org/en/latest/specifications/version-specifiers/).

if you can ignore a lot of quite legalese language from the python community, then you get to a [useful function](https://packaging.python.org/en/latest/specifications/version-specifiers/#appendix-parsing-version-strings-with-regular-expressions):

```python
import re
def is_canonical(version):
    return re.match(r'^([1-9][0-9]*!)?(0|[1-9][0-9]*)(\.(0|[1-9][0-9]*))*((a|b|rc)(0|[1-9][0-9]*))?(\.post(0|[1-9][0-9]*))?(\.dev(0|[1-9][0-9]*))?$', version) is not None
```
which will test your version:

```raw
In [3]: is_canonical('v0.0.4')
Out[3]: False
```
This would work:

```raw
In [4]: is_canonical('0.0.4')
Out[4]: True
```

tl;dr -- versions should be just numeric, although they do allow for additional stuff at the end

**Comment by Dan-RAI** [comment]
Ok, will make a new 0.0.5 release over the weekend.

**Comment by Dan-RAI** [comment]
Changed the release tag. Please let me know if this is sufficient.

**Comment by cmatKhan** [comment]
great -- thank you!

I'm having trouble with the gcc settings in the bioconda recipe. Not sure when I'm going to get back to this -- probably not until next month.
